src/cr/cube/matrix/measure.py

```diff
@@ -550,6 +555,33 @@ def _subtotal_rows(self):
         return SumSubtotals.subtotal_rows(self._base_values, self._dimensions)
 
 
+class _ShareSum(_BaseSecondOrderMeasure):
+    """Provides the share of sum measure for a matrix."""
```

Comment:
Also maybe a couple of words about _what_ these values will be? Maybe not, I'm not sure, but this seems a bit thin for a first-time reader.